autorest/azure/environments.go

```diff
@@ -108,6 +111,7 @@ var (
 		GalleryEndpoint:              "https://gallery.chinacloudapi.cn/",
 		KeyVaultEndpoint:             "https://vault.azure.cn/",
 		GraphEndpoint:                "https://graph.chinacloudapi.cn/",
+		ServiceBusEndpoint:           "https://servicebus.chinacloudapi.net/",
```

Comment:
Is this the correct TLD?

Reply:
based on this: https://msdn.microsoft.com/en-us/library/azure/dn578439.aspx, it appears that it should be `.cn`, which makes more sense. I'll update.